### ISSUE TITLE
Unable to initialise rancher2 provider after upgrading to Terraform 0.12

### ISSUE THREAD

**Issue by ishantanu** [rancher/terraform-provider-rancher2#14]
Hi,

I am unable to initialise `rancher2` provider after upgrading to terraform. Also, since `rancher2` was officially available in terraform registry, I removed the `terraform-provider-rancher2` from `~/.terraform.d/plugins`.

After this, I started to see this error:
```
Provider "rancher2" not available for installation.

A provider named "rancher2" could not be found in the Terraform Registry.

This may result from mistyping the provider name, or the given provider may
be a third-party provider that cannot be installed automatically.

In the latter case, the plugin must be installed manually by locating and
downloading a suitable distribution package and placing the plugin's executable
file in the following directory:
    terraform.d/plugins/darwin_amd64

Terraform detects necessary plugins by inspecting the configuration and state.
To view the provider versions requested by each module, run
"terraform providers".


Error: no provider exists with the given name
```

**Comment by rawmind0** [comment]
Hi @ishantanu,

By the moment this provider is not compatible with terraform 0.12 . We are working to update it asap, PR #16

**Comment by rawmind0** [comment]
Hi @ishantanu, [rancher2 provider v1.1.0](https://github.com/terraform-providers/terraform-provider-rancher2/releases/tag/v1.1.0) is just released. This release is compatible with terraform v0.12

To upgrade terraform to v0.12 in a proper way, please follow [terraform upgrade guide](https://www.terraform.io/upgrade-guides/0-12.html)

**Comment by rawmind0** [comment]
@ishantanu, please reopen issue if needed.

**Comment by Tediferous** [comment]
@rawmind0 I am still getting this same error on Terraform v0.12.0

**Comment by rawmind0** [comment]
@Tediferous, to upgrade terraform to v0.12 in a proper way, please follow [terraform upgrade guide](https://www.terraform.io/upgrade-guides/0-12.html) 

Due to a release issue (it has been solved), provider download is not working on tf v0.12. Please init it on tf v0.11.14 and upgrade to v0.12. It works in that way.

**Comment by rawmind0** [comment]
Rancher2 provider release issue has been solved on terraform 0.12. `terraform init` on tf v0.12 should now download Rancher2 provider correctly.
